project/Settings.scala

```diff
@@ -104,6 +104,7 @@ object Settings {
   lazy val serverDockerSettings = commonDockerSettings ++ List(
     mainClass in Compile := Some("org.broadinstitute.dsp.workbench.welder.server.Main"),
     packageName in Docker := "broad-dsp-gcr-public/welder-server",
+//    daemonUser in Docker := "welder-user",
```

Comment:
Did you mean to include this?

Reply:
No. It’s not working yet. I’ll try it tomorrow